odoo/addons/test_apikeys/static/tests/apikey_flow.js

```diff
@@ -72,7 +72,7 @@ tour.register('apikeys_tour_setup', {
 // deletes the previously created key
 tour.register('apikeys_tour_teardown', {
     test: true,
-    url: '/web',
+    url: '/web?debug=1',
```

Comment:
Probably used when working, but to remove ?

Reply:
same here

Reply:
Same about note then